src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/services/WorkflowService.scala

```diff
@@ -143,15 +162,24 @@ class WorkflowServiceImpl(override val workflowRepository: WorkflowRepository,
     workflowRepository.getProjectsInfo()
   }
 
-  override def runWorkflow(workflowId: Long)(implicit ec: ExecutionContext): Future[Boolean] = {
+  override def runWorkflows(workflowIds: Seq[Long])(implicit ec: ExecutionContext): Future[Boolean] = {
     val userName = getUserName.apply()
-
-    for {
-      joinedWorkflow <- workflowRepository.getWorkflow(workflowId)
-      dagInstanceJoined <- dagInstanceService.createDagInstance(joinedWorkflow.dagDefinitionJoined, userName)
-      _ <- dagInstanceRepository.insertJoinedDagInstance(dagInstanceJoined)
-    } yield {
-      true
+    workflowIds.distinct.length match {
+      case numberOfWorkflows if numberOfWorkflows == 1 =>
+        throw new ApiException(GenericError(s"More than 1 workflow has to be triggered!"))
+      case numberOfWorkflows if numberOfWorkflows > ApplicationConfig.maximumNumberOfWorkflowsInBulkRun =>
+        throw new ApiException(GenericError(
+          s"Maximum number of workflows = ${ApplicationConfig.maximumNumberOfWorkflowsInBulkRun}, for bulk run has been exceeded!"
```

Comment:
The error message seems a bit cumbersome. How about e.g. "Cannot trigger more than x workflows"?

Reply:
Ok